### DISCUSSION TITLE
Good Repos to contribute to for Hacktoberfest

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#321] [Events]
As we approach Hacktoberfest, we'd like to have a list of welcoming, positive, and credible repositories for our members who are interested in participating in Hacktoberfest.

If you know of a repository that we should consider, drop them in the discussion below and let us know why you think they're a good open source project for our members. 

We'd love to have a variety of projects for members of all different levels and backgrounds. And be sure to use our [Maintainers Checklist](https://virtualcoffee.io/member-resources/oss-maintainer-checklist/) to ensure you have all the basics.

**Comment by ghost** [comment]
Hello @BekahHW 
I'm happy to hear Hacktoberfest is coming for this year. Not coming by itself but by your work guys. So thanks a million!
I'm learning myself web development, meanwhile I try to make a tiny useful web-apps.

Here are two of my repositories  that I consider well documented and good for beginners:

[Classified-ads-48](https://github.com/bacloud14/Classified-ads-48)  
[WeatherVenue](https://github.com/bacloud14/WeatherVenue)  

Really basic stuff.

**Reply by ghost** [reply]
Please note that I fully understand if these two repos are not accepted. Code is not that much organized and is maybe poorly written. Most repositories would deserve to come first, as I'm not a web-developer at the first place/. 👍

**Reply by BekahHW** [reply]
@bacloud14 very cool. If you haven't already, go ahead and see if your repos fulfill the [checklist](https://virtualcoffee.io/member-resources/oss-maintainer-checklist/) we have up on the site.

**Reply by ghost** [reply]
Thanks @BekahHW ! Everything seem fine, but I have a question:
Does "Contributing guide" and "Code of Conduct" need to be in separate files ? because I care of these but I mention them only in README file.

**Reply by tkshill** [reply]
@bacloud14 these projects look great! thanks for sharing! I think the contributing/code of conduct stuff is alright 👍🏾

**Comment by MarieReRe** [comment]
Woo Woo! Hacktoberfest is the best! This year as Community Engineer at Harness I am leading our very first Hacktoberfest for Drone.io 🎉 We will be focusing on plugin creations as well as looking for doc updates and bug fixes! Here is our doc repo and will list more as we figure it all out! https://github.com/drone/docs

**Reply by tkshill** [reply]
@mrsantons are you still looking to open drone docs up for contributions?

**Reply by MarieReRe** [reply]
Yes we are! Also plug-in creation

**Reply by tkshill** [reply]
Let us know when you get the supporting stuff in place so we can add it to the list (contributing, CoC, issue tags, etc) 🙏🏾

**Reply by teezzan** [reply]
@mrsantons I'd like to attempt plugin creation. Can you share with me links?

**Reply by teezzan** [reply]
https://github.com/orgs/drone-plugins/repositories

**Comment by aurelieverrot** [comment]
Lucia shared StepZen repo. They are an active contributing company during HacktoberFest:
https://virtual-coffee-group.slack.com/archives/C01DYTKRYAY/p1631303197091800
Stack is GraphQL, React, Nuxt.JS (Vue)

**Reply by nickytonline** [reply]
We'll definitely have frontend and backend (Rails/Ruby) in Forem, https://github.com/forem/forem. I'm also finishing up getting our Gitpod config to work properly so that the friction to get Forem installed locally is completely removed.

I haven't checked lately, but I imagine the Protege repository, https://github.com/drewclem/protege, will have some issues. I upgraded the site to NEXT.js 11 and it revealed a bunch of lint errors, so there's that at least.

Brian Douglas from GitHub has a project called Open Sauced, https://github.com/open-sauced/open-sauced, that has a bunch of issues, most likely up for grabs for Hacktoberfest.

I'll try and think of some others.

**Reply by aurelieverrot** [reply]
Meg shared RubyForGood repo:

https://github.com/rubyforgood/rubyforgood.org
Stack: Ruby

**Comment by Cerchie** [comment]
StepZen repos:
React/GraphQL/StepZen
https://github.com/stepzen-samples/stepzen-spacex-graphql
Nuxt/GraphQL/StepZen
https://github.com/stepzen-samples/stepzen-nuxt

**Reply by nickytonline** [reply]
These look great. I haven't done much GraphQL and zero Vue. 😎

**Comment by CalvinWilkinson** [comment]
Here are a 2 projects that I just recently open sourced.  I will admit, these are not related to web development at all though.
These projects are all related to cross platform libraries for 2D game development or multi-media applications as well as processing sound.

All built with NET 5.0 and moving to cross platform as much as possible!!
I also have 3 more projects that are in this realm that I have not yet but plan on moving to open source.
A 2D particle engine, 2D game engine and a binding library for SDL2 (Simple Direct Media Layer) which can be used for cross platform media and game applications.

1. [Velaptor](https://github.com/KinsonDigital/Velaptor)
    * This is currently in _**v1.0.0-preview.1**_
2. [CASL](https://github.com/KinsonDigital/CASL)
    * This is currently in _**v1.0.0-preview.10**_

**Reply by tkshill** [reply]
@CalvinWilkinson these look good! Non web dev is welcome! Do you have any issues you might think are good for newbie OSS contributors in Velaraptor?

**Reply by CalvinWilkinson** [reply]
Sure.  My list of current issues are quite small though because I just open sourced it not too long ago.  But the issues I am putting in are starting to pile up. 

Actually @teezzan graciously offered to help out with a simple workflow issue.  

There are a couple simple ones for [CASL](https://github.com/KinsonDigital/CASL)
1. KinsonDigital/CASL#52
2. KinsonDigital/CASL#69

As usual, any improvements in code documentation and unit tests are welcome as well.  I am also looking to fine tune any documentation on documentation related to making it clear on how to contribute and the expectations on contributions from things such as coding standards to making sure that it is clear and understandable on how to contribute.  

So if anybody decides to help out, don't hesitate to let me know about any confusing areas or issues.  I want to make sure that the friction is as minimal as possible.  For example, the branching strategy I have put in place is indeed more restrictive and different then most OS projects I think but most of the more complicated stuff will be performed by me anyways.

**Comment by avneesh0612** [comment]
I have a few Next.js projects, I will link them here

1. [Audible clone](https://github.com/avneesh0612/voyagger) 

2. [ChatCube](https://github.com/avneesh0612/ChatCube) 

3. [Portfolio Template](https://github.com/avneesh0612/portfolio-template) 

4. [Voyagger ](https://github.com/avneesh0612/voyagger)

**Reply by tkshill** [reply]
these look interesting, especially voyagger 👍🏾

**Reply by avneesh0612** [reply]
Thanks @tkshill

**Comment by danieltott** [comment]
Thank you to everyone who has submitted answers here! If you'd like to get involved in Virtual Coffee's Hacktoberfest Initiative as a maintainer, be sure to sign up as a maintainer at [hacktoberfest.virtualcoffee.io](https://hacktoberfest.virtualcoffee.io)!

We'll still vet and share any repos that are shared here, but signing up as a maintainer is a great way to contribute to our efforts and work directly with some of our contributors 💪

**Comment by CalvinWilkinson** [comment]
Thanks!!
